src/libm/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -O3 -DNDEBUG")
+
+set(LIBSLEEF_SRCS sleefdp.c sleefsp.c)
+
+add_library("${TARGET_LIBSLEEF}" SHARED ${LIBSLEEF_SRCS})
+
+# Always compile sleef with -ffp-contract and log at configuration time
+message(STATUS "Using option `${STRICTMATHFLAG}` to compile libsleef")
+target_compile_options(${TARGET_LIBSLEEF} PUBLIC ${STRICTMATHFLAG})
+
+# Shared library versioning
+set_target_properties(${TARGET_LIBSLEEF} PROPERTIES VERSION ${SLEEF_VERSION_MAJOR}.${SLEEF_VERSION_MINOR} SOVERSION ${SLEEF_SOVERSION})
```

Comment:
This results in the following library and symbolic links:
libsleef.so.3.0
libsleef.so.3 -> libsleef.so.3.0
libsleef.so -> libsleef.so.3

I believe this is standard shared library versioning: using a version layers that maps to the project version (ie SLEEF 3.0) and another separate layer which is the library version (we currently use the MAJOR project version to be the library version too).

I shall add a comment explaining this.